jscomp/super_errors/super_typemod.ml

```diff
@@ -2,6 +2,22 @@ open Printtyp
 
 let fprintf = Format.fprintf
 
+let pp_component_type_not_generalizable_pre ppf =
+  fprintf ppf "@[<v>\
```

Comment:
There's an extra v box here now; was that intentional?

Reply:
Other occurrences have been moved to the outside. Let me try to remove this and see what happens.

Reply:
Checked: yes it matters to avoid squashing things.

Reply:
It ensures "Here's the original error message" is on a new line.

Reply:
@cristianoc I sent a invitation to you, feel free to merge it when you think it is ready